references/detection/train_tensorflow.py

```diff
@@ -11,8 +11,6 @@
 import datetime
 import hashlib
 import time
-from collections import deque
-from pathlib import Path
 
 import numpy as np
 import tensorflow as tf
```

Comment:
May be we also have to import wandb in the script?

Reply:
but it is on line19 ;)